sdk/nodejs/policy/policy.ts

```diff
@@ -13,37 +13,76 @@
 // limitations under the License.
 
 import * as pulumi from "@pulumi/pulumi";
-import { Resource } from "@pulumi/pulumi";
 import * as q from "@pulumi/pulumi/queryable";
+
 import { serve } from "./server";
 
+/**
+ * Arguments for constructing a new PolicyPack object.
+ */
 export interface PolicyPackArgs {
+    // Policies to associate with the policy pack.
```

Comment:
Let's make this a `/** */` doc comment style comment, since this is a library. That way, the comment will show up in intellisense when using the API in IDEs like VS Code.

Reply:
Done. I assumed it would "just work", but even then we should be consistent.